.github/composite-actions/setup-cache/action.yml

```diff
@@ -0,0 +1,27 @@
+name: "Setup Cache"
+
+inputs:
+  os:
+    description: 'OS version'
+    required: true
+  php:
+    description: 'PHP version'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get Composer cache directory
+      id: composer-cache-dir
+      shell: bash
+      run: echo "dir=$(composer config cache-dir)" >> $GITHUB_OUTPUT
+
+    - name: Cache dependencies
```

Comment:
so...

I purged the whole cache for repo, run the jobs, cache was created afterwards
![Screenshot 2023-11-03 at 19 04 21](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/2716794/0abc23d9-3959-42fd-95d3-0da12c955ae2)

Yet, we are getting this error (only on the PR, i do not see it on master branch)
https://github.com/keradus/PHP-CS-Fixer/actions/runs/6748717442/job/18347647031#step:31:3

Reply:
yep, it should store even without merge
ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/6748899599/job/18348194309?pr=7413